apps/webClient/src/views/Home/index.tsx

```diff
@@ -25,46 +20,43 @@ const Home: React.FC<Props> = ({}) => {
   const classes = useStyles();
 
   return (
-    <Box pt={16}>
-      <Grid container justify="center">
-        <Logo />
-      </Grid>
+    <>
       <Box mt={12}>
```

Comment:
If you want we should just give the logo a margin bottom so we don't have to do it on each page